env_requirements.txt

```diff
@@ -0,0 +1,50 @@
+https://repo.anaconda.com/pkgs/main/win-64/git-2.23.0-h6bb4b03_0.conda

```

Comment:
how would you install the requirements using this file?
could you add instructions to the README ?